docs/pages/modules/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Overview
+---
+
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+
+Expo Modules is a set of APIs and utils to improve the process of developing native modules for React Native and increase its capabilities. The entire ecosystem consists of the following parts, each one described in details on the dedicated pages:
+
+- [Module API](module-api) — Allows creating native modules in a modern way, using Swift and Kotlin.
+- [Autolinking](autolinking) — The mechanism that automatically links native dependencies.
+- [AppDelegate Subscribers](appdelegate-subscribers) — Allows reacting to iOS AppDelegate events.
+- [Module Config](module-config) — The module config file to configure and opt into some features.
+
+If you would like to create a new Expo module from scratch, we have prepared a dedicated command for this: `npx create-expo-module`.<br/>
```

Comment:
```suggestion
To create a new Expo module from scratch, run `npx create-expo-module`.<br/>
```

Reply:
i might change this to a heading:

```
## Create a new Expo module

The `create-expo-module` package initializes a new Expo module from scratch.

<!-- insert <TerminalBlock /> like https://docs.expo.dev/get-started/create-a-new-app/#initializing-the-project ->
```

easier to find at a glance. then i'd add a similar heading for "Using the Expo Modules API in an existing React Native library"